### ISSUE TITLE
a tutorial for distributed text deduplication

### ISSUE THREAD

**Issue by simplew2011** [dask/dask#11149]
Can you provide an example of distributed text deduplication based on dask, such as：
- https://github.com/xorbitsai/xorbits/blob/main/python/xorbits/experimental/dedup.py
- https://github.com/ChenghaoMou/text-dedup/blob/main/text_dedup/minhash_spark.py

**Comment by phofl** [comment]
Hi Thanks for opening the issue!

This sounds interesting to me. Are you interested in writing something up?

**Comment by randerzander** [comment]
It might help to take a look at NeMo Curator's Dask based [(gpu only) fuzzy dedup](https://github.com/NVIDIA/NeMo-Curator/blob/main/examples/fuzzy_deduplication.py) and [exact dedup](https://github.com/NVIDIA/NeMo-Curator/blob/main/examples/exact_deduplication.py) modules.

**Comment by simplew2011** [comment]
Yes, NeMo Curator provides a gpu-cudf implementation. Can we implement a CPU version as an advanced tutorial for dask.

**Comment by randerzander** [comment]
To clarify, Curator's exact dedup does work on CPU

**Comment by simplew2011** [comment]
Is there any progress？
